data/14.0.0.xml

```diff
@@ -5,14 +5,14 @@
     <changelog href="https://nextcloud.com/changelog/#14-0-0" />
     <whatsNew lang="en">
         <regular>
-            <item>Verify the identity of a share recipient in a video call; and 2FA with Signal &amp; Telegram</item>
-            <item>Add a note to shares, see pending &amp; deleted shares and search for files by content of comments</item>
-            <item>Accessibility improvements including dark &amp; high contrast themes and a dyslectic font</item>
+            <item>Verify identity of a share recipient in a video call; 2FA with Signal &amp; Telegram</item>
+            <item>Add a note to shares; see pending &amp; deleted shares; search for files by content of comments</item>
+            <item>Accessibility improvements including dark &amp; high contrast themes and dyslectic font</item>
         </regular>
         <admin>
-            <item>Improved GDPR compliance capabilities with terms &amp; conditions app and separate audit log</item>
-            <item>Next generation of federated, cross-cloud sharing collaboration: Federation 2.0</item>
-            <item>Swiftv3 auth support, logging to systemd, Kerberos auth to Samba &amp; multi-IdP SAML</item>
+            <item>Improved GDPR compliance with terms &amp; conditions app; separate audit log</item>
```

Comment:
the app is called terms of service

Reply:
thx!

Reply:
Ah, is being dropped anyway, cf https://github.com/nextcloud/changelog_server/pull/7#issuecomment-413681768